### ISSUE TITLE
Local Time vs Zulu for File Create Date

### ISSUE THREAD

**Issue by snethej** [jmcnamara/excel-writer-xlsx#162]
When I create a file, the date stored as the file created date (as seen in the peoperties) is 5 hours behind my current time, and my current time is 5 hours behind UTC.

For example, I just created a file.  My local time is 8:01 am (CDT). UTC is 13:01 (1:00 pm). The timestamp of the created date of the file is 3:01 am.  This would be the time in Hawaii.

Looking at the core.xml file in the doc props archive of the xlsx file, I see the following:

```xml
<dcterms:created xsi:type="dcterms:W3CDTF">2016-05-10T08:01:46Z</dcterms:created>
<dcterms:modified xsi:type="dcterms:W3CDTF">2016-05-10T08:01:46Z</dcterms:modified>
```
It appears the the local server time is being written as Zulu time, which is pushing the time back in the wrong direction.

**Comment by jmcnamara** [comment]
Thanks for the report. That's a bug.

You can workaround it for now by adding this to your program:

```perl
    $workbook->set_properties(created => [ gmtime() ]);
```

I'll fix it soon.

**Comment by jmcnamara** [comment]
I've pushed a fix to the master branch for this issue if you would like to try it.

**Comment by snethej** [comment]
I will try and get to it soon.  Thanks for the support!
<div>
</div><div>
</div><div>-------- Original message --------</div><div>From: John McNamara <notifications@github.com> </div><div>Date: 05/10/2016  5:46 PM  (GMT-06:00) </div><div>To: jmcnamara/excel-writer-xlsx <excel-writer-xlsx@noreply.github.com> </div><div>Cc: snethej <snethej@hotmail.com>, Author <author@noreply.github.com> </div><div>Subject: Re: [jmcnamara/excel-writer-xlsx] Local Time vs Zulu for File Create Date (#162) </div><div>
</div>
I've pushed a fix to the master branch for this issue if you would like to try it.

---

**Comment by jmcnamara** [comment]
Fixed in Excel::Writer::XLSX version 0.90.

Thanks for the report.
